qtranslate_core.php

```diff
@@ -1381,7 +1381,24 @@ function qtranxf_use_block($lang, $blocks, $show_available=false, $show_empty=fa
 	//if(isset($post)){
 	//	//qtranxf_dbg_echo('$post='.$post);
 	//}
-	return "<p>".preg_replace('/%LANG:([^:]*):([^%]*)%/', $language_list, $q_config['not_available'][$lang])."</p>";
+
+	$altlanguagecontent = '.</p>';
+	if ( !empty($q_config['always_show_content']) && $q_config['always_show_content'] ) { // TEMPORARY UNTIL BOOLEAN AVAILABLE TO BE EDITED IN ADMIN
+		// determine the language	
+		$altlanguage = $q_config['default_language'];
+		if ( !in_array($altlanguage, $available_languages) ) { $altlanguage = $available_languages['0']; }
+		$altlanguagecontent = trim($content[$altlanguage]); 
+		
+		// echo which language is used
+		if ( count($available_languages) == '1' ) {
+			$altlanguagecontent = ' '.__('If you want to have a look:', 'qtranslate').'</p>';
```

Comment:
Zeus, are you sure, we need this 'if'? Would not 'Showing in  ' always work? The different text here could also (or only) depend on whether available language is the default or not. For default language we could say 'The content is shown below in this site default language ', or otherwise 'The content below is shown in the first available language '. Or simply 'The content below is shown in ' and people will understand that this is simply the first in the list. What do you think?